java/src/com/ibm/streamsx/topology/internal/context/ToolkitStreamsContext.java

```diff
@@ -232,7 +232,8 @@ public FileVisitResult preVisitDirectory(final Path dir,
             public FileVisitResult visitFile(final Path file,
                     final BasicFileAttributes attrs) throws IOException {
                 Files.copy(file,
-                        targetPath.resolve(sourcePath.relativize(file)));
+                        targetPath.resolve(sourcePath.relativize(file)),
+                        StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Why is REPLACE_EXISTING needed?

It seems to imply that the same file is being copied multiple times.